### ISSUE TITLE
Devhub meta titles on Google search results probably not showing as intended

### ISSUE THREAD

**Issue by elarb** [devhub-tud/devhub#490]
![image](https://user-images.githubusercontent.com/18485279/27953899-db2b598c-630d-11e7-98ee-aacafb286b2d.png)

**Comment by JWGmeligMeyling** [comment]
Yes I noticed Google indexing our Maven repository too. Our Maven repository is not maintained by any of the Devhub programmers so I am not sure when this will be picked up.
